PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -48,6 +48,8 @@ struct HfCandidateSelectorLc {
   Configurable<bool> usePidBayes{"usePidBayes", true, "Bool to use or not the PID based on Bayesian probability cut at filtering level"};
   Configurable<double> ptPidBayesMin{"ptPidBayesMin", 0., "Lower bound of track pT for Bayesian PID"};
   Configurable<double> ptPidBayesMax{"ptPidBayesMax", 100, "Upper bound of track pT for Bayesian PID"};
+  // Combined PID options
+  Configurable<bool> useTpcAndTofPID{"useTpcAndTofPID", false, "Bool to decide how to combine TPC and TOF PID: true = both (if present, only one otherwise); false = one is enough"};
```

Comment:
```suggestion
  Configurable<bool> usePidTpcAndTof{"usePidTpcAndTof", false, "Bool to decide how to combine TPC and TOF PID: true = both (if present, only one otherwise); false = one is enough"};
```
To appear next to `usePid` and `usePidBayes` when sorted alphabetically in the Hyperloop menu.